src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,109 @@
+import styled from "styled-components";
+
+import PageContentContainer from "../ui/PageContentContainer";
+
+import {
+  breakpoints,
+  colors,
+  fonts,
+  respondFrom,
+  respondTo,
+} from "../../styles";
+
+import FoodImage from "../../assets/images/BannerImage.png";
+import FoodImageMobile from "../../assets/images/BannerImageMobile.png";
+
+const BannerMain = styled.section`
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+  padding: 22px 0;
```

Comment:
any chance this could be 24px?

Reply:
I change it to new spacing variable based on 8 px